src/subscript/sector2fluxnum/sector2fluxnum.py

```diff
@@ -0,0 +1,251 @@
+import argparse
+import sys
+import os
+import datetime
+from cwrap import open
+from ecl.grid import EclGrid
+from ecl.eclfile import EclFile, FortIO
+from subscript.sector2fluxnum import flux_obj
+from subscript.sector2fluxnum import fluxfile_obj as ffo
```

Comment:
Suggest not to alias these. It only reduces readability, code is read much more often than it is typed.